cfme/tests/physical_infrastructure/ui/test_physical_server_list_buttons.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.physical.provider.lenovo import LenovoProvider
```

Comment:
**Required**

Please organize imports, with blank line between uninstallable imports (pytest) and local.

Organized local by alphabetical, so `cfme.common` imports should be before `cfme.physical`, etc.

Reply:
ok, done.